php80/fpm/Dockerfile

```diff
@@ -35,6 +35,11 @@ RUN docker-php-ext-install bcmath mysqli pdo_mysql soap zip intl opcache xsl pcn
     && docker-php-ext-configure gd --with-freetype --with-jpeg --with-webp \
     && docker-php-ext-install gd
 
+# Install xdebug
+RUN pecl install xdebug \
+    && docker-php-ext-enable xdebug \
+    && rm -rf /tmp/pear
```

Comment:
pecl was removed from php 8, was t re-added?

Reply:
When I run a `docker build` and then a `docker run` on the built image with `php -v`, it shows xdebug installed.

![image](https://user-images.githubusercontent.com/642417/113691655-f4d49780-96cc-11eb-80fe-5806abc9fb51.png)

Leave it to the author of xdebug to fix this for us: docker-library/php#1087

Reply:
So, officially it wasn't removed, just not enabled. It explicitly required `--with-pear` it seems, which has been added to the official PHP images.